install.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/sh
```

Comment:
Consider using bash instead of sh
```suggestion
#!/usr/bin/env bash
```